### ISSUE TITLE
MultiMiner crashes when opening in MacOS

### ISSUE THREAD

**Issue by ftulio** [nwoolls/MultiMiner#280]
I installed XQartz and Mono.
Already tried to change chmod MultiMiner.win
Tried to open manually using terminal.

It keeps crashing when opening.

In attach is the report/log 
[MultiMiner.txt](https://github.com/nwoolls/MultiMiner/files/1289793/MultiMiner.txt)

**Comment by moodonis** [comment]
not much to add, but same problem here.

Exception Type:        EXC_BAD_ACCESS (SIGABRT)
Exception Codes:       KERN_INVALID_ADDRESS at 0x00000000042b27ec
Exception Note:        EXC_CORPSE_NOTIFY

**Comment by andresteves** [comment]
+1

**Comment by tommaton** [comment]
+1

**Comment by talfouzan** [comment]
+1

**Comment by nwoolls** [comment]
What is the output of the following:

I've updated the installation instructions here with a workaround:

* https://github.com/nwoolls/MultiMiner/wiki/Installation#mac-os-x

```
cd /Applications/MultiMiner.app/Contents/Resources
mono --arch=32 MultiMiner.Win.exe
```

**Comment by nwoolls** [comment]
Fixed in [v4.1.1](https://github.com/nwoolls/MultiMiner/releases/tag/v4.1.1) - thanks for taking the time to report this.
